cli/code_scanner.py

```diff
@@ -31,20 +31,31 @@
               help='Branch to scan, if not set scanning the default branch',
               type=str,
               required=False)
+@click.option('--monitor',
+              is_flag=True,
+              default=False,
+              help="When specified, the scan results will be sent to Cycode platform and results will be parsed as "
+                   "vulnerabilities and violations.",
+              type=bool,
+              required=False)
 @click.pass_context
-def scan_repository(context: click.Context, path, branch):
+def scan_repository(context: click.Context, path, branch, monitor):
```

Comment:
give it a boolean name

Reply:
what is the different between this to all other flags that we have in code - verbose/show-secrets/soft-fail?